mixins/form.jsx

```diff
@@ -6,10 +6,11 @@ module.exports = {
   mixins: [Navigation],
   getInitialState: function() {
     var amount = "";
+    var frequency = "single";
     if (this.props.queryString) {
       amount = this.props.queryString.amount || "";
+      frequency = this.props.queryString.frequency || frequency;
```

Comment:
maybe do a check on that queryString.frequency if it is equal to `single` or `monthly` in case someone put crap in there.